assets/js/custom.js

```diff
@@ -1,15 +1,100 @@
-var controllerHome = (function(jQuery) {
-    var adjustHomeLink = function() {
-        if (document.location.hostname === 'localhost') {
-            jQuery('.navbar-brand').attr('href', '/');
-            jQuery('#nav-0').attr('href', '/');
-        }
-    };
-
-    return {
-        init: adjustHomeLink
-    };
-
-}(jQuery));
+var controllerHome = (function (jQuery) {
+  const CONSENT = 'consentGiven';
+  const consentBanner = jQuery("#consent-banner");
+  const acceptCookiesButton = jQuery("#accept-cookies");
+  const denyCookiesButton = jQuery("#deny-cookies");
+  const gtagId = "{{ site.gtagId }}";
```

Comment:
I don't think this works in the JS, you have to init the `const gtagId = G-3V6VPT445S`